prime-angular-frontend/src/app/modules/paper-enrolment/pages/demographic-page/demographic-page.component.html

```diff
@@ -84,11 +84,44 @@
 
       <app-address-form [form]="formState.physicalAddress"
                         [showManualButton]="true"
-                        [showAddressFields]="showAddressFields"></app-address-form>
+                        [showAddressFields]="showAddressFields(formState.physicalAddress.value)"></app-address-form>
+
+      <app-page-subheader2>
+        <ng-container appPageSubheader2Summary>Provide any additional addresses.</ng-container>
```

Comment:
- Will check the space. I also wasn't comfortable with the 2 '+Add' lines. 
- I can add a divider between the 2 buttons just so they don't look like they're grouped together
- I liked that as it shows its different but can put them on same width

With that being said after checking with Pilar and the fact that this will only be used by admins on rare occasions we decided not to spend more time on it, hence the aesthetics weren't a priority.

Reply:
Yeah the buttons are not too bad having them like that since low use case.

Understand aesthetics are a priority but you added additional code/work and made it look not inline with how the rest of the page works.

Looking into how the address form works it uses row so having this external row just adds unnecessary spacing and margins 
![image](https://user-images.githubusercontent.com/59669288/160505552-f79d4afc-84f9-43c3-9781-54b3c5cb39f5.png)

There is no need to have these two divs and added styling, when a ng-container will suffice.

Reply:
Removed the divs,

- Space is now the same for enrollee's addresses and additional addresses titles and their corresponding fields.
- Added a divider between the 2 `Add` buttons
- Width now matches, minus the X mark width